test/nn/dense/test_mincut_pool.py

```diff
@@ -15,3 +16,18 @@ def test_dense_mincut_pool():
     assert adj.size() == (2, 10, 10)
     assert -1 <= mincut_loss <= 0
     assert 0 <= ortho_loss <= 2
+
+    if is_full_test():
+        batch_size, num_nodes, channels, num_clusters = (2, 20, 16, 10)
+        x = torch.randn((batch_size, num_nodes, channels))
+        adj = torch.ones((batch_size, num_nodes, num_nodes))
+        s = torch.randn((batch_size, num_nodes, num_clusters))
+        mask = torch.randint(0, 2, (batch_size, num_nodes), dtype=torch.bool)
```

Comment:
Lets reuse values defined in lines 8-12, similar to what was done in https://github.com/pyg-team/pytorch_geometric/pull/5754/files.